examples/cfg.ron

```diff
@@ -23,6 +23,21 @@
         "i_is_enter": 4,
     },
 
+    layer_profiles: {
```

Comment:
I'm wondering whether the "off" profile fields make these a bit verbose/awkward.
Intuitively, when I try to imagine what I'd want a profile to look like, I only think about which layers should be on.
I'd expect the rest of the layers to be off.

Would that also work with the workflow you had in mind? 
Turning off all layers, then applying the layers in the profile.

Reply:
Eh yeah you're probably right where we only have one profile at a time. Again I was thinking of having multiple profiles loaded, but we can just do one if you think that is better. I was kind of thinking something where you have a base profile for an app and you could add extra profiles. But if you think that "extra" profile should just be entirely encompassing of all functionality for the user, I think that makes sense

Reply:
Yeah. I think it'll be simpler / more intuitive. 
Though, we can definitely iterate on this again in the future if we find my suggestion lacking.